README.md

```diff
@@ -6,39 +6,72 @@ Query API for the Tezos blockchain
 ## Running Conseil
 
 Conseil has two entry points:
-- `src/main/Conseil.scala` runs a server exposing a RESTful API.
-- `src/main/Lorre.scala` is a background process that runs on loop and writes Tezos blockchain data to database.
+- `src/main/tech/cryptonomic/Conseil/Conseil.scala` runs a server exposing a RESTful API.
+- `src/main/tech/cryptonomic/Conseil/Lorre.scala` is a background process that runs on loop and writes Tezos blockchain data to database.
 
 ### Warning 
 
 The Conseil server should be run behind a proxy such as Nginx with TLS enabled through something like LetsEncrypt. Futhermore, [HTTP Strict Transport Security](https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security) and [Certification Authority Authorization](https://en.wikipedia.org/wiki/DNS_Certification_Authority_Authorization) are highly recommended!
 
 ### Prerequisites
-- JDK
-- Scala
-- SBT
+
+Development
+- JDK (>7.x)
+- Scala (>2.11.x)
+- SBT (>1.2.6)
+- A database supported by Typesafe Slick, e.g. Postgres
+
+Deployment
+- JRE (>7.x)
 - A database supported by Typesafe Slick, e.g. Postgres
 
-### Compile
+### Building, packing and deploying Conseil
+
+#### Compile
+
+Run the following command from the root directory of the repository:
 
 `sbt compile`
   
-### Package
+#### Package
+
+Run the following command from the root directory of the repository:
  
 `sbt assembly`
+
+#### Deploy
+
+After the package step, check the SBT output for the path of the generated fat JAR file. Copy this file to the desired deployment area.
   
-### Run
+### Running Locally
 
 Run the fat JAR with the JRE:
 
 `java -Dconfig.file={path to custom config file} -cp {path to fat JAR} tech.cryptonomic.conseil.Conseil`
 
+And..
+
+`java -Dconfig.file={path to custom config file} -cp {path to fat JAR} tech.cryptonomic.conseil.Lorre`
```

Comment:
This change hasn't been implemented into the prod and staging nodes, does that mean it's unstable or should we update the nodes to reflect the changes made since deployment?
if we need to implement it in staging to test if it's stable let's test it before making the change.  The README should reflect stable settings.

Reply:
As far as I can tell, yes, the current `staging` configuration is unstable, as I would expect `lorre` to fail at startup requiring to specify what network to run on.

If it's not so, I am puzzled

Reply:
@leenpaws, this change has been in master for quite sometime now.